test/terraform/main.tf

```diff
@@ -0,0 +1,13 @@
+
+// DynamoDB Table to store Terraform State Lock
+resource "aws_dynamodb_table" "dynamodb-terraform-state-lock" {
```

Comment:
I'm not sure if we need to update the oidc IAM access [here](https://github.com/newrelic/infrastructure-agent/blob/ce6dffc48132e3f7393d5ecaae4800c6206f73d4/test/automated/terraform/fargate/main.tf#L92) or somewhere [here](https://github.com/newrelic/fargate-runner-action/blob/main/terraform/modules/infra-ecs-fargate/main.tf)

Reply:
Indeed.
I have added it here https://github.com/newrelic/fargate-runner-action/pull/34

and added to this repo here https://github.com/newrelic/infrastructure-agent/pull/1868/commits/dedb40a781e4393b7c8ce3b38c82838d9e74c31d

Reply:
But I'm thinking that this should be conditional... moving it to a draft to give it a proper thought